web-app/js/portal/details/LayerControlPanel.js

```diff
@@ -19,37 +19,50 @@ Portal.details.LayerControlPanel = Ext.extend(Ext.Container, {
             this.items.push(layerSelector, { xtype: 'spacer', height: 10 });
         }
 
-        this.items.push(this._newOpacitySlider());
+        this.opacitySlider = new Portal.common.LayerOpacitySliderFixed({
+            layer: this.layer,
```

Comment:
I see that we're now passing the listeners as part of the config, @pmbohm. So I'd like to see this component creation moved back into it's own function as it was. We no longer need to assign `this.opacitySlider` because it isn't referenced anywhere else.